### ISSUE TITLE
Please add Traditional Chinese to language file

### ISSUE THREAD

**Issue by GoogleCodeExporter** [WhiteNeo/wiiflow#153]
```
Please add Traditional Chinese to language file, I have translated it(for
wiiflow R42)and uploaded this file.
```

Original issue reported on code.google.com by `lyr032...@gmail.com` on 9 Apr 2010 at 4:12

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
dup with Issue 149
```

Original comment by `noboo...@gmail.com` on 10 Apr 2010 at 8:59

**Comment by GoogleCodeExporter** [comment]
```
sorry im not meantion about, but traditional chinese include in my patch.
```

Original comment by `noboo...@gmail.com` on 10 Apr 2010 at 9:03

**Comment by GoogleCodeExporter** [comment]
```
Well, I'm willing to add this, but it's all Chinese to me :P

So this is really different then the [Chinese] language in language.ini?
```

Original comment by `r-...@live.com` on 29 May 2010 at 10:17

**Comment by GoogleCodeExporter** [comment]
```
1.Yes.
2.Yes,Both of them are really different.
 - Simplified Chinese (GB2312)--->this language is used in China.
 - Traditional Chinese (Big5) --->this language is used in Taiwan.
Hope to add this file on next version.

```

Original comment by `lyr032...@gmail.com` on 3 Jun 2010 at 9:41

**Comment by GoogleCodeExporter** [comment]
```
Added to the language file, it's available since version 2.0
```

Original comment by `r-...@live.com` on 22 Jun 2010 at 7:49
- Changed state: **Fixed**
